src/py/flwr/client/mod/secure_aggregation/secaggplus_mod.py

```diff
@@ -492,20 +509,19 @@ def _collect_masked_input(
 def _unmask(
     state: SecAggPlusState, configs: ConfigsRecord
 ) -> Dict[str, ConfigsRecordValues]:
-    log(INFO, "Client %d: starting stage 3...", state.sid)
+    log(INFO, "Node %d: starting stage 3...", state.nid)
 
-    active_sids = cast(List[int], configs[Key.ACTIVE_SECURE_ID_LIST])
-    dead_sids = cast(List[int], configs[Key.DEAD_SECURE_ID_LIST])
+    active_nids = cast(List[int], configs[Key.ACTIVE_NODE_ID_LIST])
+    dead_nids = cast(List[int], configs[Key.DEAD_NODE_ID_LIST])
     # Send private mask seed share for every avaliable client (including itclient)
```

Comment:
What does `itclient` mean?

Reply:
Great catch. It's a typo - should be `itself`.